frontend/ui/otamanager.lua

```diff
@@ -208,6 +208,13 @@ function OTAManager:checkUpdate()
 end
 
 function OTAManager:fetchAndProcessUpdate()
+    if Device:hasOTARunning() then
+        UIManager:show(InfoMessage:new{
+            text = _("Download already scheduled. You'll be notified when it is complete"),
```

Comment:
Something like this?
```suggestion
            text = _("Download already scheduled. You'll be notified when it's ready."),
```